### ISSUE TITLE
how do you set this up ?

### ISSUE THREAD

**Issue by PieterDes** [nitaybz/homebridge-delay-switch#19]
Hi,

I've searched the web, the issues and I think I did everything correct but can't get it to work.

What I tried : 
created a delay switch for 1 min.
Created an automation that says to turn on the light when the delay switches OFF. 

triggered delay switch manually

Nothing happens.

I did this in the native home app.

Anyone an idea what I am doing wrong ?

Thanks a lot !

**Comment by nitaybz** [comment]
please share your config

**Comment by PieterDes** [comment]
Hi,
config : 

I have a Broadlink rm pro and some generic 433mhz outlet switches. 

Delay timers created for 1 min and 5 min. 

Switching the outlet via homekit with a button works perfectly. 

Switching outlet on when delay switch is off doesn't work at all. I do see the timer on and off in the console but nothing is triggered. 

If you want my config.json file please let me know.

Thanks a lot for the help.

**Comment by nitaybz** [comment]
try to work with the extra motion sensor... set automation to trigger when motion sensor detects motion

**Comment by PieterDes** [comment]
Hi,

also did this. I retested this because of your question but still nothing happens. 

I get the message on my iPad that the motion sensor detected movement but it does not trigger the automation.

I really have no clue what is going on.

**Comment by nitaybz** [comment]
Any other automations work for you?

**Comment by PieterDes** [comment]
Yes, I've set up an automation to switch of the same light when I am leaving home and it works. 

Also just now I've added an automation to switch on those lights at a certain time and it also works.

This is in my config for the delay :
{
         "accessory": "DelaySwitch",
         "name": "Delay 1min",
         "disableSensor": false,
         "delay": 60000
       }

**Comment by nitaybz** [comment]
Can you share a screenshot of automations

**Comment by PieterDes** [comment]
Ok,

just retested this with a smart wifi socket and a Sonoff and that worked. 
Now I added the RF socket (switched on by Broadlink RM PRO) and it also worked. 

So...no idea why or how but my automations are the same but now it seems to work.

I did another restart of homebridge but no idea if that helped...

Anyway, I'm sorry for all my questions but very grateful for your help.

I'll close the thread !
